lib/doorkeeper-jwt/config.rb

```diff
@@ -25,6 +25,10 @@ def build
           @config
         end
 
+        def use_application_secret(use_application_secret)
+          @config.instance_variable_set('@use_application_secret', use_application_secret)
```

Comment:
Line is too long. [90/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.